SU-HSDO/suhumsci
# feat(shs-3886): adding utility class

# READY FOR REVIEW

## Summary
Small PR for adding an `.hb-clear` utility class to be used as needed.

## Urgency
medium

## Steps to Test
1. Edit a layout like the event layout display: https://west.suhumsci.loc/admin/structure/types/manage/hs_event/display/default/layout
2. Add the `.hb-clear` class to the Contact block close to the bottom of the page. 
3. Note that the block now clears floating items above it. 
https://west.suhumsci.loc/events/salinas-history-race-and-resilience-agricultural-city

![Screenshot from 2024-01-21 23-40-05](https://github.com/SU-HSDO/suhumsci/assets/2343957/206043da-374b-4bfb-a31b-c4edc5b9250e)

Note that once this is deployed, the documentation will need to be updated as noted in the Clickup Ticket.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
